app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/NightscoutUploader.java

```diff
@@ -209,6 +211,11 @@ private void populateV1APIBGEntry(JSONArray array, BgReading record) throws Exce
             json.put("unfiltered", record.usedRaw() * 1000);
             json.put("rssi", 100);
             json.put("noise", record.noiseValue());
+            json.put("xDrip_raw", record.raw_data);
+            json.put("xDrip_filtered", record.filtered_data);
+            json.put("xDrip_calculated_value", record.calculated_value);
+            json.put("xDrip_age_adjusted_raw_value", record.age_adjusted_raw_value);
+            json.put("sysTime", format.format(record.timestamp));
```

Comment:
Why do we need this uploaded?
Especially as most of it is duplicates.

This will blow up network use and make NightScout more expensive.